src/subtitle/cue.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * Represents the direction in which the cue text should be written.
+ */
+export enum CueDirection {
```

Comment:
This type does not exist in the iOS SDK, where does this come from?

As far as I can tell this should be [BMPVttVertical](https://cdn.bitmovin.com/player/ios/3/docs/Type%20Definitions.html#/c:BMPVttVertical.h@T@BMPVttVertical)

The values associated with the enum cases should match the native representation.

Reply:
I tried to explain the rationale behind this decision [here](https://github.com/bitmovin/bitmovin-player-react-native/pull/14#discussion_r948039545). But still, if you think they don't make sense, I'll change them back.

Reply:
Also @rolandkakonyi, the values are already mapping the native representation:

- `HORIZONTAL` -> `VttVertical.unset` (It's stated in [the docs](https://cdn.bitmovin.com/player/ios/3/docs/Classes/VttProperties.html#/c:@M@BitmovinPlayer@objc(cs)BMPVttProperties(py)vertical) that `unset` means `horizontal` so I used the latter because it's way more meaningful)
- `LEFT_TO_RIGHT` -> `VttVertical.leftToRight`
- `RIGHT_TO_LEFT` -> `VttVertical.rightToLeft`

Reply:
Cues support has been dropped for now, see [commit](https://github.com/bitmovin/bitmovin-player-react-native/pull/14/commits/1599353362e30aa2471b60d7ddcff02d88bf8b2c).